basis/html/templates/chloe/compiler/compiler.factor

```diff
@@ -74,17 +79,38 @@ SYMBOL: string-context?
 
 ERROR: tag-not-allowed-here ;
 
+CONSTANT: self-closing-tags {
```

Comment:
I think the original idea was two-fold.  1) sets were created after sequences, so duplication.  2) implied complexity of member? is O(n), but in? could be faster.

I agree, it would better to unify some of the container words and implement cursors and other improvements, and hope to do some of that as part of the larger work for 0.99, but have wanted to get a good 0.98 release first.